frontend/javascripts/libs/error_handling.js

```diff
@@ -93,6 +93,11 @@ class ErrorHandling {
       });
     }
 
+    window.addEventListener("unhandledrejection", event => {
+      // Create our own error for unhandled rejections here to get additional information for [Object object] errors in airbrake
+      this.notify(Error("Unhandled Rejection"), { originalError: event.reason });
```

Comment:
Wow, nice, this should be quite helpful! I think airbrake also shows the most recent requests so that we should be able to make the connection between rejected promises and requests.